yaroslav.nazarenko/HW_7/image.rb

```diff
@@ -0,0 +1,21 @@
+require "open-uri"
```

Comment:
You don't need to use open-uri, also Please fix all rubocop issues below so I can review the code